tests/test_package_generation.py

```diff
@@ -124,46 +110,31 @@ def test_pip_installable(
     )
 
 
-@pytest.mark.parametrize("funder", ["", "STFC"])
+@pytest.mark.parametrize("funder", ["", "STFC", "UKRI", "Wellcome Trust"])
 def test_optional_funder(
-    tmp_path: pathlib.Path, generate_package: typing.Callable, funder: str
+    funder: str, default_config_with: typing.Callable, generate_package: typing.Callable
 ) -> None:
     """Test specifying funder or not in package generation."""
-    config = {
-        "github_owner": "test-user",
-        "project_short_description": "description",
-        "project_name": "Cookiecutter Test",
-        "funder": funder,
-    }
+    config = default_config_with("funder", funder)
```

Comment:
```suggestion
    config = default_config_with(funder=funder)
```
Required if we use suggestion in https://github.com/UCL-ARC/python-tooling/pull/566/files#r2224973492